docs/var.tfvars-doc.md

```diff
@@ -255,4 +255,6 @@ This variable is used to set the default Container Network Interface (CNI) netwo
 
 ```
 cni_network_provider       = "OpenshiftSDN"
+clusterNetwork_CIDR        = "10.128.0.0/14"
+serviceNetwork             = "172.30.0.0/16"
```

Comment:
We can rename variables to `cluster_network_cidr` and `service_network` (at all places)